Logger.Tests/LogFactoryTests.cs

```diff
@@ -5,7 +5,44 @@ namespace Logger.Tests
 {
     [TestClass]
     public class LogFactoryTests
-    {
-        
+    {
+
+        [TestMethod]
+        public void Logger_Returns_Null()
+        {
+            LogFactory logFactory = new LogFactory();
+
+            BaseLogger testLogger = logFactory.CreateLogger(nameof(testLogger));
+
+            Assert.IsNull(testLogger);
+        }
+
+        [TestMethod]
+        public void Logger_Returns_Configured_Logger()
+        {
+            string testPath = Path.GetRandomFileName();
+
+            LogFactory logFactory = new LogFactory();
+            logFactory.ConfigureFileLogger(testPath);
+
+            BaseLogger testLogger = logFactory.CreateLogger(nameof(testLogger));

```

Comment:
```suggestion
            BaseLogger testLogger = logFactory.CreateLogger(nameof(LogFactoryTests));

```
This should be the name of the class creating the logger.